engine/tests/loaders/test_coronate_situational.py

```
@@ -0,0 +1,92 @@
+
+import pytest
+
+from tangl.mechanics.progression import LinearGrowthHandler
+from tangl.mechanics.progression.challenges import StatChallenge, resolve_challenge
+
+
+def _domain():
+    root = Path(__file__).resolve().parents[3] / "worlds" / "coronate_the_regent"
+    if str(root) not in sys.path:
+        sys.path.insert(0, str(root))
+    import coronate_the_regent.domain as d  # noqa: WPS433
```

Ruff (0.15.12)

[warning] Invalid rule code in `# noqa`: WPS433

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)